app/src/main/java/org/simple/clinic/bp/BloodPressureRepository.kt

```diff
@@ -115,4 +115,14 @@ class BloodPressureRepository @Inject constructor(
   }
 
   fun measurement(uuid: UUID): Single<BloodPressureMeasurement> = Single.fromCallable { dao.getOne(uuid) }
+
+  fun markBloodPressureAsDeleted(bloodPressureMeasurement: BloodPressureMeasurement): Completable {
+    return Completable.fromAction {
+      val deletedBloodPressureMeasurement = bloodPressureMeasurement.copy(
+          deletedAt = Instant.now(clock),
+          syncStatus = SyncStatus.PENDING)
```

Comment:
Can you please confirm with backend if we need to also bump the `updateAt` timestamp?

Reply:
Good catch; we are supposed to do it. I have fixed the PR.